content/doc/book/pipeline/getting-started.adoc

```diff
@@ -0,0 +1,226 @@
+integrating continuous delivery pipelines into Jenkins. Pipeline provides an
+extensible set of tools for modeling simple-to-complex delivery pipelines "as
+code" via the Pipeline DSL.
+footnoteref:[dsl,link:https://en.wikipedia.org/wiki/Domain-specific_language[Domain-Specific Language]]
+
+This section will introduce some of the key concepts to Jenkins Pipeline and
+help introduce the basics of defining and working with Pipelines inside of a
+running Jenkins instance.
+
+== Prerequisites
+
+To use Jenkins Pipeline, a relatively recent version of Jenkins must be used with the  appropriate plugins installed:
+
+* Jenkins 2.x or later (older versions back to 1.642.3 will work but are not
+  recommended)
+* Pipeline plugin
+footnoteref:[pipeline, link:https://plugins.jenkins.io/workflow-aggregator[Pipeline plugin]]
+
+To learn how to install and manage plugins, consult <<../managing/plugins#, Managing Plugins>>.
+
+== Defining a Pipeline
+
+Pipeline Script is written with
+link:http://groovy-lang.org/[Groovy].
+The relevant bits of
+link:http://groovy-lang.org/semantics.html[Groovy syntax]
+will be introduced as necessary in this document, so while an understanding of
+Groovy is helpful, it is not required to use Pipeline Script.
+
+A basic Pipeline can be created in either of the following ways:
+
+* By entering a script directly in the Jenkins web UI.
+* By creating a `Jenkinsfile` which can be checked into a project's source
+  control repository.
+
+The syntax for defining a Pipeline with either approach is the same, but while
+Jenkins supports entering Pipeline Script directly into the web UI, it's
+generally considered best practice to define the Pipeline in a `Jenkinsfile`
+which Jenkins can then load directly from source control.
+footnoteref:[scm, https://en.wikipedia.org/wiki/Source_control_management]
+
+
+=== Defining a Pipeline in the Web UI
+
+To create a basic Pipeline in the Jenkins web UI, follow these steps:
+
+* Click *New Item* on Jenkins home page.
+
+image::new-item-selection.png["Click *New Item* on the Jenkins home page", role=center]
+
+* Enter a name for your Pipeline, select *Pipeline* and click *OK*.
+
+image::new-item-creation.png["Enter a name, select *Pipeline*, and click *OK*", role=center]
+
+* In the *Script* text-area, enter a Pipeline script and click *Save*.
+
+image::hello-world-script.png["In the *Script* text-area, enter a Pipeline script and click Save", role=center]
+
+* Click *Build Now* to run the Pipeline.
+
+image::build-now.png["Click *Build Now* to run the Pipeline", role=center]
+
+
+* Click *#1* under "Build History" and then click *Console Output* to see the
+  full output from the Pipeline.
+
+image::hello-world-console-output.png["Click *Console Output* for the Pipeline", role=center]
+
+The example above shows a successful run of a basic Pipeline created in the Jenkins
+web UI, using two valuable steps.
+
+[pipeline]
+----
+// Script //
+node { // <1>
+    echo 'Hello World' // <2>
+}
+----
```

Comment:
Indeed, I think we need to play with the warnings a bit more. There is some documentation which refers explicitly to Pipeline Script.

I'm also not including any Declarative Pipeline in this pass since I expect to merge this before Declarative is made a dependency of the `workflow-aggregator` plugin